molgenis-platform-integration-tests/src/test/resources/conf/molgenis.properties

```diff
@@ -0,0 +1,6 @@
+#Molgenis server properties used at dev time
+db_user=molgenis
+db_password=molgenis
+db_uri=jdbc\:postgresql\://localhost/molgenis_test?reWriteBatchedInserts=true&autosave=CONSERVATIVE
```

Comment:
Makes sense to set them programmatically in the PostgresRepository?
https://jdbc.postgresql.org/documentation/publicapi/org/postgresql/ds/common/BaseDataSource.html#setReWriteBatchedInserts-boolean-